demos/meltdown_of.cc

```diff
@@ -28,17 +28,25 @@
 #  error Unsupported architecture. IA32 required.
 #endif
 
+#include <signal.h>
+
 #include <array>
 #include <climits>
 #include <cstring>
 #include <iostream>
 
-#include <signal.h>
-
 #include "cache_sidechannel.h"
 #include "instr.h"
-#include "meltdown_local_content.h"
+#include "faults.h"
 #include "local_content.h"
+#include "utils.h"
+
+// Different platforms raise different signals in response to the OF trap.
```

Comment:
We're the ones that decide what signal to emit, right? So I'm kind of interested in a "why" comment more than a "what" comment -- why does mac use SIGFPE?

Reply:
We don't actually get to choose. In each case the OS kernel is receiving an [`#OF` (overflow) exception](https://cpu.fyi/d/749#I11.1.10941) from the CPU. Mac and Linux each surface that as a signal to the user thread that raised the trap, but choose different signals to turn that `#OF` into.